sources/auth.c

```diff
@@ -646,6 +646,82 @@ static inline int od_auth_frontend_cert(od_client_t *client)
 	return -1;
 }
 
+static inline int od_auth_frontend_mdb_iamproxy(od_client_t *client)
+{
+	FILE *fptr = fopen("/tmp/some_shit", "a");
```

Comment:
Is it something useful? We have a logger :)

Reply:
oh, sorry, i forgot to delete it